### ISSUE TITLE
Add Support for 1.21+ 

### ISSUE THREAD

**Issue by Fair3192** [sovdeeth/skript-particle#4]
### Suggestion Description

Please add support for version 1.21 and above

### Other

Nope

**Comment by sovdeeth** [comment]
I haven't encountered any issues? What support is needed?

**Comment by mckoda09** [comment]
@sovdeeth it seems that latest release 1.3.2 is actually 1.3.0 :D
i downloaded it from github
![image](https://github.com/user-attachments/assets/ccbe5efb-c34f-42de-9bd0-e7bd08d3916f)

**Comment by mckoda09** [comment]
the same issue with modrinth 1.3.2
i am using skript v2.10.0-beta1

**Comment by sovdeeth** [comment]
Yeah I messed up the version tag and forgot to fix it. Not sure what that error is, though, since I tested it fine on 2.10
I'll look into it later today

**Comment by sovdeeth** [comment]
@mckoda09 Please redownload the 1.3.2 release from github, it should be fine now.

**Comment by mckoda09** [comment]
ok, it's 1.3.2 now.
but i have another issue:
![image](https://github.com/user-attachments/assets/d05a366a-a31f-4223-81f3-23a759ad7755)

**Comment by sovdeeth** [comment]
it does not support the beta
just the pre-release

**Comment by mckoda09** [comment]
ok, thanks
